packages/proxy/lib/http/util/prerequests.ts

```diff
@@ -80,7 +80,8 @@ export class PreRequests {
   requestTimeout: number
   pendingPreRequests = new StackMap<PendingPreRequest>()
   pendingRequests = new StackMap<PendingRequest>()
-  sweepInterval: ReturnType<typeof setInterval>
+  sweepIntervalId: NodeJS.Timeout
```

Comment:
NodeJS.Timeout is an object, not an ID

```suggestion
  sweepInterval: NodeJS.Timeout
```

Reply:
Woah, did not know this. Fixed.

Reply:
https://github.com/cypress-io/cypress/pull/25209/commits/ba9339f6fd8624d396897c4ab2bba37c093c7639